draft-ietf-quic-multipath.md

```diff
@@ -1092,20 +1092,19 @@ Path Identifier:
   MAY be specified, even if they are not yet in use over a path.
 
 Path Status Sequence Number:
-: A variable-length integer specifying the sequence number assigned for
+: A variable-length integer specifying the per-path sequence number assigned for
   this PATH_STANDBY frame. The sequence number space is shared with the
-  PATH_AVAILABLE frame and the sequence
-  number MUST be monotonically increasing generated by the sender of
-  the PATH_STANDBY frame in the same connection. The receiver of
-  the PATH_STANDBY frame needs to use and compare the sequence numbers
-  separately for each Path ID.
+  PATH_AVAILABLE frame and monotonically increasing values MUST be used.
+
+As the sequence space is per-path, the same sequence number may be received
+for different Path Identifier values and as such the receiver of
+the PATH_STANDBY frame needs to use and compare the sequence numbers
+separately for each Path ID.
 
```

Comment:
This requirement is already stated above in the definition of the field and we should not write it down twice:

 > A variable-length integer specifying the per-path sequence number assigned for
 > this PATH_STANDBY frame. The sequence number space is shared with the
 > PATH_AVAILABLE frame and monotonically increasing values MUST be used.

This paragraph is just an additional note to make things completely obvious. It's not necessarily needed but given this is a late change and it might confuse people, I think it's better to be crystal clear.

Reply:
Specifically, I don't want to say "as the sequence is per path". I would be OK with something simple like:

Path Status Sequence Number:
: A variable-length integer specifying the sequence number assigned for
  this PATH_STANDBY frame. The sequence number space is shared with the
  PATH_AVAILABLE frame and monotonically increasing values MUST be used
  for PATH_STANDBY or PATH_AVAILABLE frame related to the same path-ID.

My point is that we should not force implementations to have a send counter per path, since having a global send counter works just fine.